spell/spell-crafter-mainnet-workflow.md

```diff
@@ -189,6 +189,9 @@ Repo: https://github.com/makerdao/spells-mainnet
   * [ ] Ensure local tests PASS against deployed spell run via the deploy script
   * [ ] Push auto-generated `add deployed spell info` commit
 * Cast spell on a newly created Tenderly Testnet
+  * [ ] `export TENDERLY_USER='<ORG_SLUG>'` (e.g.: dewiz-xyz)
+  * [ ] `export TENDERLY_PROJECT='makerdao'`
+  * [ ] `export TENDERLY_ACCESS_KEY='<ACCESS_KEY>'` (from the **account level**, not from the organization level) 
```

Comment:
I don't think checklist is a good place to store env variables and setup instructions:
- Setup only needs to happen once while every checklist item suppose to be followed every single time
- Code of the tenderly script lays in a different repository so it's harder to maintain two places in sync

Instead, you can link a readme section that already explains that:
```suggestion
* Cast spell on a newly created Tenderly Testnet (see [setup instructions](https://github.com/makerdao/spells-mainnet?tab=readme-ov-file#cast-to-tenderly))
```



Alternatively, you can submit a PR into `makerdao/spells-mainnet` to move this setup instructions into a more visible place (eg inside [relevant folder](https://github.com/makerdao/spells-mainnet/tree/master/scripts/cast-on-tenderly))